.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.idea/
```

Comment:
BTW, this is often placed in user-s config as well, not project-s. After all, we can't really always include ALL of the editors to each project, right? Try moving the ignore line to `~/.config/git/ignore`, I think it'll work. Or if you're on Windows, https://help.github.com/en/articles/ignoring-files

Reply:
y